doc/user/OPTIONS.md

```diff
@@ -182,3 +182,13 @@ For example the screenshot filename is configured as `{app}/{model}_{n}.png` by
 consecutive screenshots are going to be saved as `F3D/hello_1.png`, `F3D/hello_2.png`, `F3D/hello_3.png`, ...
 
 Model related variables will be replaced by `no_file` if no file is loaded and `multi_file` if multiple files are loaded using the `multi-file-mode` option.
+
+## HDRI Caches
+
+When using HDRI related options, F3D will create and use a cache directory to store related data in order to speed up rendering.
+These cache files can be safely removed at the cost of recomputing them on next use.
+
+The cache directory location is as follows, in order, using the first defined environnement variables:
+ - Windows: `%LOCALAPPDATA%\f3d`
+ - Linux: `${XDG_CACHE_HOME}/f3d`,`${HOME}/.cache/f3d`
+ - macOS: `${XDG_CACHE_HOME}/f3d`,`${HOME}/Library/Caches/f3d`
```

Comment:
@Meakk is that the expected behavior on macOS ?

Reply:
I need to double check but I don't think `XDG_CACHE_HOME` is used tbh

Reply:
my reference is this so post: https://stackoverflow.com/questions/3373948/equivalents-of-xdg-config-home-and-xdg-data-home-on-mac-os-x but its not clear to me if these variables are set or not on macOS.

Reply:
Just checked, they are not